UPGRADE-3.x.md

```diff
@@ -4,3 +4,10 @@ UPGRADE 3.x
 ## Deprecated Admin class
 
 The `Admin` class is deprecated. Use `AbstractAdmin` instead.
+
+## Deprecated template fallback mechanism
+
+The method that gets templates in the Twig extension loaded a default template
+when the one specified could not be found. You can no longer rely on that and
```

Comment:
A linebreak at 120 chars like PHP would be a minimum.

And especially on MD file, it's sad and harder to read to linebreak a line for two words..